pkg/ipam-node/handlers/handlers.go

```diff
@@ -115,12 +116,20 @@ func validateReq(req paramsGetter) error {
 	return nil
 }
 
+func setDefaultsToParams(params *nodev1.IPAMParameters) *nodev1.IPAMParameters {
+	if params.PoolType == nodev1.PoolType_POOL_TYPE_UNSPECIFIED {
```

Comment:
Can it ever be unspecified? Looking at the CNI plugin code, it looks like we always set it to a proper value.

Reply:
The CNI plugin will always set the value, but I think on the server side we should not do any assumptions about client's behavior.